views/app.jade

```diff
@@ -1,15 +0,0 @@
-doctype html
-html(manifest=o.htmlWebpackPlugin.files.manifest)
-  head
-    meta(charset="utf-8")
-    title= o.htmlWebpackPlugin.options.title || 'Simplenote'
-    //- iOS webviews do not support indexedDB!
-    //- meta(name="apple-mobile-web-app-capable" content="yes")
```

Comment:
@beaucollins I almost feel like this is an important-enough it of knowledge to keep in the template so that nobody accidentally tries to add it back in. how do you feel about it? can we even write comments in ejs?